tests/rados/test_mon_pg_warn.py

```diff
@@ -87,15 +89,12 @@ def mon_pg_warn_health_warn(should_contain: bool):
             mon_pg_warn_health_warn(should_contain=True)
 
             if test_type == "pg_autoscale":
-                # enable PG autoscaling
-                mon_obj.remove_config(
-                    section="global", name="osd_pool_default_pg_autoscale_mode"
-                )
-                rados_obj.set_pool_property(
-                    pool=pool_name, props="pg_autoscale_mode", value="on"
-                )
+                # PG autoscale WARN mode cannot be removed as it
+                # would also remove the health warning
+                # Autoscale PG in the pool to 32 using pg_num pool property
+                rados_obj.set_pool_property(pool=pool_name, props="pg_num", value=32)
                 rados_obj.set_pool_property(
-                    pool=pool_name, props="pg_num_max", value=256
+                    pool=pool_name, props="pg_num_min", value=32
```

Comment:
I believe even this would not be needed, as the PGs would be sutoscaled to to 32 from 1.

Reply:
Autoscaling is set to WARN state throughout, so manual scale up necessary